### ISSUE TITLE
Vouch doesn't set correct validator_index when calling /eth/v1/validator/beacon_committee_subscriptions`

### ISSUE THREAD

**Issue by sauliusgrigaitis** [attestantio/vouch#75]
We testing [Grandine](https://github.com/sifraitech/grandine) Beacon Node API with Vouch and we found that Vouch doesn't set correct `validator_index` when calling `/eth/v1/validator/beacon_committee_subscriptions`:

```
"level":"trace","service":"client","impl":"http","id":"4c4079f1","address":"localhost:9301","endpoint":"/eth/v1/validator/beacon_committee_subscriptions","body":"[{"validator_index":"0","slot":"482059","committee_index":"12","committees_at_slot":"25","is_aggregator":false},{"validator_index":"0","slot":"482074","committee_index":"18","committees_at_slot":"25","is_aggregator":false},{"validator_index":"0","slot":"482073","committee_index":"10","committees_at_slot":"25","is_aggregator":false},...
```
Grandine doesn't like it as it checks that the validator is actually in the committee.

**Comment by mcdee** [comment]
I tested this against a different beacon node and it appears to be sending the correct validator indices:

```
[{"validator_index":"985","slot":"499121","committee_index":"0","committees_at_slot":"1","is_aggregator":false},{"validator_index":"913","slot":"499128","committee_index":"0","committees_at_slot":"1","is_aggregator":false},...]
```

which suggests that it is an issue with a different call.  Please could you run vouch with the following additional configuration:

```
eth2client:
  log-level: trace
```

and send over the log file?  This should give a bit more information to help track down the issue.

**Comment by mcdee** [comment]
Fixed in #94
